docs/tutorials/data_sources/pytorch_dataset_tutorial.md

```diff
@@ -0,0 +1,164 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.17.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# PyTorch dataset tutorial
```

Comment:
Is it possible to have a more descriptive title? Maybe something like "How to use a Torchvision dataset in the Grain pipeline"?